keg_storage/cli.py

```diff
@@ -0,0 +1,113 @@
+import click
+import functools
+from flask.cli import with_appcontext
+import humanize
+
+from keg_storage.backends.base import FileNotFoundInStorageError
+from keg_storage import utils
+
+
+def handle_not_found(func):
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            func(*args, **kwargs)
+        except FileNotFoundInStorageError as e:
+            raise click.FileError(
+                e.filename,
+                hint="Not found in {}.".format(str(e.storage_type))
+            )
+
+    return wrapper
+
+
+def add_cli_to_app(app, cli_group_name):
```

Comment:
@bladams as we discussed last week this might be an option for you: http://click.pocoo.org/6/commands/#merging-multi-commands